README.md

```diff
@@ -1 +1,87 @@
-# Technical-Task-Challenges
\ No newline at end of file
+# Technical Task Challenges
+
+Virtual Coffee's Technical Task Challenges are created/aggregated by VC members to help other VC members gain practical experience in how to use technical skills in real-world scenarios.
+
+### Benefits
```

Comment:
**chore:** [`MD001` - Heading levels should only increment by one level at a time](https://github.com/DavidAnson/markdownlint/blob/main/doc/md001.md)

```suggestion
## Benefits
```